mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -80,13 +80,8 @@ def case_open(self):
 
         for case in self:
             values = {'active': True}
-
-            stages = self.env['mgmtsystem.action.stage']
-            values['stage_id'] = stages.search([
-                ['is_ending', '=', False],
-                ['is_starting', '=', False]
-            ]).id
-
+            values['stage_id'] = self.env.ref(
+                'mgmtsystem_action.stage_open').id
```

Comment:
The Stages list is editable.
I can scrap all the default stages and create new ones.
In that case, your fix wouldn't work, since there is no Stage with that XML ID.

But the flaw is in the design: "Open" and "Close" buttons worked well on closed lists of States.
When we evolve to configurable Stages, they are no longer useful, since we can perfectly have several "open" Stages in our Kanban.
The solution would be to drop those buttons, and have the user move the Action the the exact Stage they want.